frontend-thea-aa/src/lib/components/EmissionEvents.svelte

```diff
@@ -7,23 +7,27 @@
 	let amountToShow = $state(3)
 </script>
 
-<h3>Emission Events</h3>
+<div class="max-w-4xl mx-auto py-6">
+  <h3 class="text-l font-bold mb-4 text-gray-800">Emission Events</h3>
```

Comment:
this file is pretty similar to the contaminated sites file, maybe consider making a shared component. If it doesn't work that's fine but probably worth considering